web_monitoring/db.py

```diff
@@ -163,6 +168,22 @@ def from_env(cls):
 Alternatively, you can instaniate Client(user, password) directly.""")
         return cls(email=email, password=password, url=url)
 
+    def request(self, method, url, data=None, raw_response=False, **kwargs):
```

Comment:
Reading through this, I just noticed that it might be nice to handle `self._api_url` here, too, e.g:

```py
if not url.startswith('http://') and not url.startswith('https://'):
    url = f'{self._api_url}{url}'
```

Reply:
Done